sphinx/source/members/constitution.rst

```diff
@@ -0,0 +1,64 @@
+Constitution
+============
+
+The :term:`constitution` defines the set of rules and conditions (as a Lua script) that members must follow for their proposals to be accepted. For example, in a CCF network governed by `4` members, a strict majority constitution would only execute proposals once `3` members (`4/2 + 1`) have voted for that proposal.
+
+Votes for the proposal are evaluated by the constitution's ``pass`` function. If the `pass` function returns ``true``, the vote is passed, and its consequences are applied to the KV in a transaction.
+
+Examples of constitution include (see :ref:`Models` for further details):
```

Comment:
Remove this "(see XXX)" - its a few lines below, and the items already link to their sub-items.

Reply:
I don't think they link to the sub-items in the `Models` section, do they?